docs/src/pages/HomePage.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import CountUp from 'react-countup';
+import Typewriter from 'typewriter-effect';
+import { Github } from '@brightlayer-ui/icons-mui';
+import Mail from '@mui/icons-material/Mail';
+import { SharedAppBar } from '../layout';
+import { Theme } from '@mui/system';
+import { PADDING } from '../shared/constants';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import { listOfAllBluiReactPackages } from '../__configuration__/listOfAllBluiReactPackages';
+import CirclesImage from '../assets/circles.svg';
+
+const CONTENT_WIDTH = 800;
+
+const styles = {
+    pageRoot: {
+        background: (theme: Theme): string =>
+            `url('${CirclesImage}') no-repeat center/200%, ${theme.palette.primary.main}`,
+        height: '100%',
+        color: 'primary.contrastText',
+    },
+    content: {
+        maxWidth: CONTENT_WIDTH,
+        margin: (theme: Theme): string => `${theme.spacing(9)} auto`,
+        padding: `${PADDING}px`,
```

Comment:
Instead of **padding** you can just use **p**.

Reply:
The PADDING constant is set to 24, but `p` only takes the "unit size", like 3 instead of 24